traefik.sample.toml

```diff
@@ -586,6 +586,14 @@
 #
 # keepAlive = 10
 
+# Force the use of the task's hostname in the backend server address even in a 
```

Comment:
I think we should be a bit more elaborative here and describe the default behavior. I'm thinking of something like the following:

_By default, a task's IP address (as returned by the Marathon API) is used as backend server if an IP-per-task configuration can be found; otherwise, the name of the host running the task is used.
The latter behavior can be enforced by enabling this switch._

Please also copy the description into the `docs/toml.md` file.